changelog.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+#### KMD Nova
+
+- Department and Caseworker classes added.
+- Ability to set security unit, department and caseworker on cases.
+- Better tests for cases.
+
+### Fixed
+
+- Security unit not set properly on Nova cases.
+
+
```

Comment:
https://github.com/DavidAnson/markdownlint and I agree that this superfluous blank line should be removed:

```suggestion
```